server/@types/express/index.d.ts

```diff
@@ -41,27 +41,10 @@ declare module 'express-session' {
     prisonerListSortOptions: string
     sessionListSortOptions: string
 
+    prisonerContexts: PrisonerContexts
+
     pageFlowHistory: PageFlow
     pageFlowQueue: PageFlow
-    // Induction related objects held on the session
-    inductionDto: InductionDto
```

Comment:
Rather than holding all these objects on the session, we now hold them as optional properties on the PrisonerContext